packages/editor/src/components/header/index.js

```diff
@@ -143,7 +143,7 @@ function Header( {
 				/>
 				<PostViewLink />
 
-				<ZoomOutToggle />
+				{ isWideViewport && <ZoomOutToggle /> }
```

Comment:
Sorry for the late review.

Isn't this `isLargeViewport`? Otherwise, it won't match the visibility of the device preview button.

The device preview dropdown appears when the browser width is `783px` or more, while the zoom out toggle appears when the browser width is `960px` or more:


https://github.com/user-attachments/assets/48c4eeef-3e6b-4074-a05d-ba9ade005225

Reply:
I'm not so sure zoom out and device preview need to behave exactly the same. I would worry more about: is the viewport large enough for zoom out to be a good experience?

Reply:
If the zoom out mode requires a wide viewport to work properly, shouldn't that be the case in device preview as well? Either way, I would expect device preview and zoom out mode to follow consistent viewport rules.